foca/api/register_openapi.py

```diff
@@ -88,3 +97,24 @@ def register_openapi(
         logger.info(f"API endpoints specified in '{spec.path_out}' added.")
 
     return app
+
+
+def _mergeSpecs(original, *args, **kwargs):
+    # first element of the list is the main file
+    original_data = ConfigParser.parse_yaml(original)
```

Comment:
`original_data` is perhaps not the best name, because you are changing it over the course of the function. Just call it `specs`.